MapboxNavigation-SPM.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -51,26 +51,26 @@
         "repositoryURL": "https://github.com/mapbox/mapbox-events-ios.git",
         "state": {
           "branch": null,
-          "revision": "e4ded10f5b9594374416f9396d5b540f8c491bbe",
-          "version": "1.0.3"
+          "revision": "1df12e2514513acdf87841cde1f98fd8e414461e",
+          "version": "1.0.4"
         }
       },
       {
         "package": "MapboxMaps",
         "repositoryURL": "https://github.com/mapbox/mapbox-maps-ios.git",
         "state": {
           "branch": null,
-          "revision": "bedc49744970602ebc10c6de3f8a2b4cc6f084a6",
-          "version": "10.0.0-rc.8"
+          "revision": "3557fa41b15e465852b38b93cc00b52fc8b9d1aa",
+          "version": "10.0.0-rc.9"
         }
       },
       {
         "package": "MapboxNavigationNative",
         "repositoryURL": "https://github.com/mapbox/mapbox-navigation-native-ios.git",
         "state": {
           "branch": null,
-          "revision": "ebe110ef2252a38cff625b12f3584d1a312e2579",
-          "version": "66.0.1"
+          "revision": "7c8ae18aa21bea2b4e801f0b6aab6a2a21d4aec1",
+          "version": "66.0.0"
```

Comment:
Doesn't look right. I guess, there should be `66.0.3`

Reply:
Or rather `67.0.0` when it's out.

Reply:
SPM doesn't let me update to `66.0.3` in case when Mapbox Maps `v10.0.0-rc.9` is used. Will update once Navigation Native `67.0.0` is available.

Reply:
Updated to Navigation Native `67.0.0`.